src/test/java/com/meilisearch/sdk/SearchRequestTest.java

```diff
@@ -0,0 +1,31 @@
+package com.meilisearch.sdk;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SearchRequestTest {
+
+	@Test
+	void getQuery() {
+		SearchRequest classToTest = new SearchRequest("This is a Test");
```

Comment:
I don't think that the default values should be handled by the SDK. I think that ideally, the query should be built only with the values that the user specifies, so in this case, a query "This is a test" shouldn't create the query `?q=This%20is%20a%20Test&offset=0&limit=20&attributesToRetrieve=*&cropLength=200&matches=false` but just `?q=This%20is%20a%20Test` and let the rest be handled by MeiliSearch itself. What do you think?

I know this is not an issue form these tests, but rather the SDK itself 👍 Just would like to have your opinion on it

Reply:
I just included this test as an example for a unit test :) The unit test folder looked a bit empty :D

Reply:
Indeed, quite a bit empty haha